### ISSUE TITLE
Seed vault cover pic needs credits and background story 

### ISSUE THREAD

**Issue by clemsos** [theatlasofdata/theatlasofdata.earth#28]
https://en.m.wikipedia.org/wiki/Svalbard_Global_Seed_Vault

That database is melting already

**Comment by clemsos** [comment]
Credits : Pic by Mari Tefre / Svalbard Globale frøhvelv - CC BY-ND 2.0

from Flickr https://www.flickr.com/photos/landbruks-_og_matdepartementet/4186766565

**Comment by clemsos** [comment]
@margocargo any refs here?

**Comment by margocargo** [comment]
@clemsos yep. i showed my students this TedTalk to present the initiative in Svalbard : https://www.youtube.com/watch?v=Uwl012o8P7I
(i'm not especially a fan of this format but it presents well the place).
The association that runs these initiatives is Crop Trust : https://www.croptrust.org/about-us/
I will dig more about the financial aspect of this place too :https://www.mediapart.fr/journal/international/220711/spitzberg-qui-finance-la-banque-de-graines

**Comment by margocargo** [comment]
The background story needs to present the climate change consequences for this place but also the paradox in the donators list, is it ok if i take this stance ?
https://www.infogm.org/6057-svalbard-pour-humanite-ou-pour-semenciers
https://www.croptrust.org/about-us/donors/

**Comment by clemsos** [comment]
@margocargo Cool! Je me demandais si il existait une liste des greniers et aussi une liste de ce qu'ils contiennent

**Comment by margocargo** [comment]
@clemsos yes, pour svalbard la database c'est : https://www.nordgen.org/sgsv/index.php?app=data_unit&unit=sgsv

**Comment by clemsos** [comment]
@margocargo Super. Ce serait pas mal de comprendre comment les graines ont été sélectionnées, essayer de voir les raisons scientifiques (ou politiques)

**Comment by clemsos** [comment]
https://www.ecowatch.com/doomsday-vault-norway-2632955972.amp.html that's quite the story

**Comment by clemsos** [comment]
What will be an interesting scientific term related to the existence of seed vaults? A word that could start a chapter out of this topic.

"Biodiversity banking" is a pretty amazing term  https://en.wikipedia.org/wiki/Biodiversity_banking

Can think of multiple art/design exploration for it
